src/Controls/GlobeControls.js

```diff
@@ -675,17 +686,40 @@ class GlobeControls extends THREE.EventDispatcher {
         }
     }
 
-    ondblclick(event) {
-        if (this.enabled === false || currentKey) { return; }
-        this.player.stop();
-        const point = this.view.getPickingPositionFromDepth(this.view.eventToViewCoords(event));
-        const range = this.getRange(point);
-        if (point && range > this.minDistance) {
-            return this.lookAtCoordinate({
-                coord: new Coordinates('EPSG:4978', point),
-                range: range * 0.6,
-                time: 1500,
-            });
+    updateMousePosition(event) {
+        coords.copy(this.view.eventToViewCoords(event));
+    }
+
+    updateTravelInProperties(event) {
+        if (this.states.TRAVEL_IN.enable) {
+            if (event && event.previous) {
+                // Remove previous travel in event listener
+                this.view.domElement.removeEventListener(event.previous, this._onTravelIn, false);
+            }
+            // Add new updated travel in event listener
+            this.view.domElement.addEventListener(this.states.TRAVEL_IN.trigger, this._onTravelIn, false);
+        } else {
+            this.view.domElement.removeEventListener(this.states.TRAVEL_IN.trigger, this._onTravelIn, false);
+        }
```

Comment:
instance `StateControls` with `view.domElement` parameter to handling mouse and keyboard events.

Reply:
I did so.